README.md

```diff
@@ -70,43 +70,141 @@ The table below summarizes the results of running various ML models through our
+| aten.addmm.default             | ✅       |      78 |
+| aten.arange.default            | ✘        |       4 |
+| aten.bmm.default               | ✅       |      24 |
+| aten.cat.default               | ✘        |       2 |
+| aten.clamp.default             | ✅       |      32 |
+| aten.clone.default             | ✅       |      50 |
+| aten.convolution.default       | ✘        |       1 |
+| aten.div.Tensor                | ✘        |      12 |
+| aten.expand.default            | ✅       |      50 |
+| aten.floor.default             | ✘        |       2 |
+| aten.gelu.default              | ✅       |      12 |
+| aten.mul.Tensor                | ✅       |      82 |
+| aten.native_layer_norm.default | ✅       |      25 |
+| aten.permute.default           | ✅       |      48 |
+| aten.relu.default              | ✅       |       4 |
+| aten.rsub.Scalar               | ✘        |      10 |
+| aten.select.int                | ✘        |       1 |
+| aten.sigmoid.default           | ✅       |       1 |
+| aten.slice.Tensor              | ✘        |      12 |
+| aten.sub.Tensor                | ✅       |      36 |
+| aten.t.default                 | ✅       |      78 |
+| aten.transpose.int             | ✅       |      15 |
+| aten.unsqueeze.default         | ✅       |       1 |
+| aten.view.default              | ✅       |     283 |
 #### Llama
-| aten ops               | status   |   count |
-|:-----------------------|:---------|--------:|
-| aten.arange.start      | ✘        |       1 |
-| aten.embedding.default | ✅       |       1 |
-| aten.unsqueeze.default | ✅       |       1 |
+| aten ops              | status   |   count |
+|:----------------------|:---------|--------:|
+| aten._to_copy.default | ✘        |       1 |
+| aten.mm.default       | ✅       |       1 |
+| aten.t.default        | ✅       |       1 |
+| aten.view.default     | ✅       |       2 |
 #### BERT
 | aten ops                       | status   |   count |
 |:-------------------------------|:---------|--------:|
 | aten._softmax.default          | ✅       |      24 |
-| aten._to_copy.default          | ✅       |       1 |
+| aten._to_copy.default          | ✘        |       1 |
 | aten.add.Tensor                | ✅       |      74 |
 | aten.addmm.default             | ✅       |     145 |
 | aten.bmm.default               | ✅       |      48 |
 | aten.clone.default             | ✅       |      99 |
-| aten.div.Tensor                | ✅       |      24 |
+| aten.div.Tensor                | ✘        |      24 |
 | aten.embedding.default         | ✅       |       3 |
 | aten.expand.default            | ✅       |      96 |
 | aten.gelu.default              | ✅       |      24 |
 | aten.mul.Tensor                | ✅       |       1 |
 | aten.native_layer_norm.default | ✅       |      49 |
 | aten.permute.default           | ✅       |      96 |
-| aten.rsub.Scalar               | ✅       |       1 |
+| aten.rsub.Scalar               | ✘        |       1 |
```

Comment:
really can't support?

Reply:
Thanks for noticing this. I fixed it with a better check for valid page sizes https://github.com/tenstorrent/pytorch2.0_ttnn/pull/53/commits/a6aa0c2f9744d203b673ba41f36ea9f348d2cbe2. rsub and div should be converted for BERT again.